### PR TITLE
added htmlproofer link check

### PR DESCRIPTION
**Pull request by ljsinclair**
Just the htmlproofer link check

### CODE REVIEW

**Review by gthrone25 — CHANGES_REQUESTED** [review]
Looks like this is missing the bash script to run? I ran bundle and nothing was generated with that name

**Comment by gthrone25** [comment]
> Looks like this is missing the bash script to run? I ran bundle and nothing was generated with that name

@ljsinclair ?

**Comment by ljsinclair** [comment]
Fixed. How much to keep this dumb mistake between us? >.<

File: README.md [hunk 1]
```diff
@@ -183,6 +183,27 @@ For larger content edits or for changes to the theme, you'll likely need to clon
...
+
+## Broken link checker
+
+* html-proofer has been added to `Gemfile`
+* load for the first time on a repository by running `bundle` at the command line.
+
+Then run the broken link checker using the batch file:
+
+```
+bash check-links.sh
```
Comment (gthrone25) [inline README.md:200]: @ljsinclair I followed the instructions above and get the following error:

> bundler: failed to load command: htmlproofer (/usr/local/bin/htmlproofer)

advice?

Comment (gthrone25) [reply README.md:200]: Nevermind I see it. This check is for x86 architecture only and won't work on amd (mac). Not so sure we should go this route now...

> /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.13.3-arm64-darwin/lib/nokogiri/extension.rb:7:in `require_relative': dlopen(/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.13.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle, 0x0009): tried: '/Library/Ruby/Gems/2.6.0/gems/nokogiri-1.13.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle' (mach-o file, but is an incompatible architecture (have (arm64), need (x86_64h))) - /Library/Ruby/Gems/2.6.0/gems/nokogiri-1.13.3-arm64-darwin/lib/nokogiri/2.6/nokogiri.bundle (LoadError)

Comment (ljsinclair) [reply README.md:200]: Oh, I'm sorry about that. I had no idea that'd occur :(

Comment (ljsinclair) [reply README.md:200]: The python one still works, it's just an alternate method

Comment (ljsinclair) [reply README.md:197]: added processor types to the 2x broken link checks to clarify what's going on

**Comment by ljsinclair** [comment]
closing this because it turns out htmlproofer only runs on x86 systems.

**Pull request closed without merging**